src/main/java/com/minecolonies/coremod/colony/managers/ColonyPackageManager.java

```diff
@@ -92,59 +87,28 @@ public void setLastContactInHours(final int lastContactInHours)
     @Override
     public Set<EntityPlayerMP> getSubscribers()
     {
-        final Set<EntityPlayerMP> set = new HashSet<>(oldSubscribers);
-        set.addAll(subscribers);
-        return set;
+        return closeSubscribers;
     }
 
     @Override
     public void updateSubscribers()
     {
         final World world = colony.getWorld();
-        // If the world or server is null, don't try to update the subscribers this tick.
+        // If the world or server is null, don't try to update the closeSubscribers this tick.
         if (world == null || world.getMinecraftServer() == null)
         {
             return;
         }
 
-        // Add owners
-        world.getMinecraftServer().getPlayerList().getPlayers()
-                .stream()
-                .filter(colony.getPermissions()::isSubscriber)
-                .forEach(subscribers::add);
-
-        //  Add nearby players
-        for (final EntityPlayer o : world.playerEntities)
-        {
-            if (o instanceof EntityPlayerMP)
-            {
-                @NotNull final EntityPlayerMP player = (EntityPlayerMP) o;
-
-                if (player.connection.networkTickCount < 5)
-                {
-                    continue;
-                }
-
-                final double distance = player.getDistanceSq(colony.getCenter());
-                if (distance < MAX_SQ_DIST_SUBSCRIBER_UPDATE
-                        || (oldSubscribers.contains(player) && distance < MAX_SQ_DIST_OLD_SUBSCRIBER_UPDATE))
-                {
-                    // Players become subscribers if they come within 16 blocks of the edge of the colony
-                    // Players remain subscribers while they remain within double the colony's radius
-                    subscribers.add(player);
-                }
-            }
-        }
-
-        if (subscribers.isEmpty())
+        if (closeSubscribers.isEmpty())
         {
             if (ticksPassed >= TICKS_HOUR)
             {
                 ticksPassed = 0;
                 lastContactInHours++;
                 colony.markDirty();
             }
-            ticksPassed++;
+            ticksPassed += 100;
```

Comment:
Make this a constant, so that on the other place where we delay it for 100 ticks, we always delay it for the same quantity

Reply:
doing that in the ticking PR, kinda belongs there :D